qiita_pet/static/js/networkVue.js

```diff
@@ -349,10 +349,11 @@ Vue.component('processing-graph', {
         vm.updateGraph();
       })
         .fail(function(object, status, error_msg) {
-          $('#run-btn').attr('disabled', false);
```

Comment:
Shouldn't this be returned only if the submission fails?

Reply:
Nop, it should be reenabled either way as the button is reused on the same page to submit multiple times. In fact, that was the issue: if a user added and ran a workflow the button stayed disabled and with the "Submitting!" text.

Reply:
Got it, thanks!